src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -1412,6 +1412,13 @@ def load_arguments(self, _):
         c.argument('ppg_type', options_list=['--type', '-t'], help="The type of the proximity placement group. Allowed values: Standard.")
         c.argument('tags', tags_type)
 
+    with self.argument_context('ppg create', min_api='2021-11-01') as c:
+        c.argument('zone', zone_type)
```

Comment:
Same as https://github.com/Azure/azure-cli/pull/23167#discussion_r917722814
```suggestion
    with self.argument_context('ppg create') as c:
        c.argument('zone', zone_type, min_api='2021-11-01')
```